api/src/main/java/io/minio/MinioClient.java

```diff
@@ -1019,7 +1011,7 @@ private HttpResponse executeDelete(String bucketName, String objectName, Map<Str
     updateRegionCache(bucketName);
     HttpResponse response = execute(Method.DELETE, BucketRegionCache.INSTANCE.region(bucketName),
                                     bucketName, objectName, null,
-                                    queryParamMap, null, null, 0);
+                                    queryParamMap, null, null, -1);
```

Comment:
> this might not be true for 'executeDelete()' when we implement Multi object delete API. Change this to executeDelete() itself taking the content length.

Multi Delete is a POST and not DELETE http operation. I think we should keep it like that until we implement the new multi delete API.